content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Running Automated Tasks with a CronJob
-min-kubernetes-server-version: v1.8
+min-kubernetes-server-version: v1.21
```

Comment:
Not sure if we need to change this ...

Reply:
Previous versions of Kubernetes don't support the GA CronJob API. We could add some text to signpost people using older Kubernetes to read docs for the version they're running.

Reply:
This page is only about CronJob in batch/v1? All old versions are removed?

Reply:
The example manifest is a `batch/v1`. We _can_ mention that there is an older beta API, and that's does feel like a good idea.
My approach to that would be to tell people that if they're using the older API, to check the documentation for an older Kubernetes release (eg v1.20).

Reply:
Added info at the top.